src/routes/(console)/project-[region]-[project]/functions/function-[function]/domains/add-domain/verify-[domain]/+page.svelte

```diff
@@ -62,6 +64,13 @@
                     page.params.domain
                 );
                 verified = domainData.nameservers.toLocaleLowerCase() === 'appwrite';
+            } else if (!isNewDomain && isCloud) {
+                const domain = await sdk.forConsole.domains.updateNameservers(domainId);
+                verified = domain.nameservers === 'Appwrite';
```

Comment:
I think we should use the same check as above and also convert the text to lowercase